ugh:
```
2024-07-11T18:00:15.7098432Z 2024/07/11 18:00:15 INFO running step "hello world c"
2024-07-11T18:00:15.7580483Z 2024/07/11 18:00:15 WARN + '[' -d /home/build ]
2024-07-11T18:00:15.7580952Z 2024/07/11 18:00:15 WARN + cd /home/build
2024-07-11T18:00:15.7581577Z 2024/07/11 18:00:15 WARN + cat
2024-07-11T18:00:15.7590759Z 2024/07/11 18:00:15 WARN + gcc -o hello-c hello.c
2024-07-11T18:00:15.7670533Z 2024/07/11 18:00:15 WARN hello.c:1:10: fatal error: stdio.h: No such file or directory
2024-07-11T18:00:15.7671643Z 2024/07/11 18:00:15 WARN     1 | #include <stdio.h>
2024-07-11T18:00:15.7672179Z 2024/07/11 18:00:15 WARN       |          ^~~~~~~~~
2024-07-11T18:00:15.7672706Z 2024/07/11 18:00:15 WARN compilation terminated.
```

--- a/gcc.yaml
+++ b/gcc.yaml
@@ -249,6 +249,10 @@ subpackages:
           done
 
 test:
+  environment:
+    contents:
+      packages:
+        - glibc-dev
   pipeline:
     - name: Check basic usage of top level & libexec binaries
       runs: |

